client.go

```diff
@@ -371,6 +370,9 @@ func (c *client) ListFiles(dir string) ([]string, error) {
 
 		filenames = append(filenames, filepath.Join(dir, info.Name()))
 	}
+
+	c.logger.Logf("found %d files: %s", len(infos), fileInfoString(filenames))
```

Comment:
```suggestion
	c.logger.Logf("found %d files: %s", len(infos), strings.Join(filenames, ", "))
```

Reply:
I'd use strings.Join() and remove `fileInfoString()`

Reply:
Yep, updated to strings.Join().